src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -181,10 +196,34 @@ const AuthoritiesSearch = ({
     return newSelectedRows;
   };
 
+  const getSelectAllRowsState = () => {
+    if(!selectAll) {
+      const newSelectedRows = {};
+      authorities.forEach((item) => {
```

Comment:
This can also be done in a shorter way:
```
const newSelectedRows = authorities.filter(item => !!item.id).reduce((acc, item) => {
  return {
    ...acc,
    [item.id]: item,
  };
}, {});
```